src/pages/ReviewWrite/ReviewWrite.js

```diff
@@ -0,0 +1,201 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import StarRatings from 'react-star-ratings';
+import { flexSet } from '../../styles/mixin';
+import { API } from '../../config';
+
+const ReviewWrite = () => {
+  const [content, setContent] = useState('');
+  const [star, setStar] = useState();
+  const [reviewfiles, setReviewFiles] = useState([]);
+
+  const changeRating = (newRating, name) => {
+    setStar(newRating);
+  };
+
+  const handleChange = e => {
+    const {
+      target: { value },
+    } = e;
```

Comment:
이것 보다는 아래가 더 깔끔할 것 같네요!
```suggestion
  const { value } = e.target; 
```

Reply:
수정했습니다 😄